system/View/Parser.php

```diff
@@ -124,6 +127,9 @@ public function render(string $view, ?array $options = null, ?bool $saveData = n
         if ($saveData) {
             $this->data = $this->tempData;
         }
+
+        $output = $this->decorateOutput($output);
```

Comment:
Is there any reason to apply decorators to `renderString()` as well? Probably not, but worth asking.

Reply:
That's a good question that I don't have a good answer to. It's not used in the framework anywhere I don't believe, so I'm not sure what it actually gets used for. I'm thinking probably no for now, and we'll see if anyone needs it.

Reply:
I think it is for handling "snippets" like `<div class="{class"">{foo}</div>`. I've never used it myself but my hunch is that decorators don't belong and would only make that confusing. I like your "wait and see" approach.